env.yml

```diff
@@ -7,47 +7,41 @@ channels:
   - defaults
 dependencies:
   - python>=3.9, <3.10
-  - bandit                                                    # (dev)
-  - black                                                     # (dev)
-  - blas
+  - pip
+  - blas>=2.0, <3
   - bottleneck>=1.2.1, <2
   - dotmap
   - fastparquet
-  - flake8                                                    # (dev)
-  - gh                                                        # (dev)
   - graphviz
-  - ipdb                                                      # (dev)
   - ipykernel
-  - isort                                                     # (dev)
   - jupyter
   - jupyterlab
   - matplotlib
   - numba>=0.46.0, <1
   - numexpr>=2.6.8, <3
   - numpy>=1.21.0, <2
-  - opencv
   - pandas>=1.3, <2
-  - pip
   - plotly
-  - poetry-core
-  - pre-commit                                                # (dev)
   - pyarrow
   - pydotplus
   - pymysql
-  - pytest                                                    # (dev)
-  - pytest-cov                                                # (dev)
   - pyyaml
   - scikit-learn
   - scipy
   - seaborn
-  - setuptools_scm
-  - sphinx                                                    # (dev)
-  - tox                                                       # (dev)
   - xarray>=0.12.3, <1
-  - pip: # Dependencies to install from pypy with `pip`.
-    - datajoint>=0.13, <1
-    - tox
-    - git+https://github.com/datajoint-company/datajoint-utilities.git
-    - git+https://github.com/Technologicat/pyan.git           # (dev)
-
-# note: dev packages can be separated out in a `env_devs.yml` file in future
+  - bandit # ....................... (dev)
+  - black # ........................ (dev)
+  - flake8 # ....................... (dev)
+  - ipdb # ......................... (dev)
+  - isort # ........................ (dev)
+  - pre-commit # ................... (dev)
+  - pytest # ....................... (dev)
+  - pytest-cov # ................... (dev)
+  - sphinx # ....................... (dev)
+  - tox # .......................... (dev)
+  - pip:
+      - datajoint>=0.13.6, <1
+      - git+https://github.com/datajoint-company/datajoint-utilities.git
+      - git+https://github.com/Technologicat/pyan.git
+      - opencv-python
```

Comment:
hey @iamamutt , is there a particular reason to install open-cv via pip from pypi rather than from conda-forge?

Also I think you took out the `gh` package - I was using this for some github cli stuff I found useful

Reply:
@jkbhagatio I added back `gh`. I thought it was something I added on my own. 

As for `opencv-python`. `pypi` doesn't have `opencv`, but `opencv-python`, and it's the official version (https://opencv.org/opencv-python-is-now-an-official-opencv-project/). Conda has `opencv`, but I couldn't find out what the build options were to assess the differences between that and the pypi version. (e.g., no ffmpeg, contrib packages, etc..).

I could put `opencv` in the `env.yml` file, and `opencv-python` in the `pyproject.toml file`, but an install of the aeon package will trigger both packages to be installed if installing from conda then pip, which will lead to issues.